### PR TITLE
specify branch=master when looking up dependencies

### PR DESCRIPTION
**Pull request by nathanleiby**
if this is specified, it goes through a simpler codepath in Catapult
that avoids hitting github.

### CODE REVIEW

File: dashboards/scripted_services.js [hunk 1]
```diff
@@ -1163,7 +1163,7 @@ var createDependencyPanel = (function() {
       appendStyles();
       $.ajax({
         method: "GET",
-        url: config.catapultURL + "/applications/" + app + "/dependencies?format=map",
+        url: config.catapultURL + "/applications/" + app + "/dependencies?format=map&branch=master",
```
Comment (xavi-) [inline dashboards/scripted_services.js]: Branch is a bit of a misnomer.  It needs to build a build-id.

Comment (xavi-) [reply dashboards/scripted_services.js]: Which you should be able to get from `appMap[env + "--" + app]`, iirc.

Comment (nathanleiby) [reply dashboards/scripted_services.js]: @xavi- updated to specify exact build ID.

image

--

Didn't see your 2nd comment, so I threaded through the build ID slightly differently. What do you think, or prefer a simpler way?

Comment (xavi-) [reply dashboards/scripted_services.js]: This way makes sense to me.

**Pull request merged**